src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -314,4 +123,13 @@ public boolean hasCapability(Capability<?> capability, EnumFacing facing) {
 
         return super.hasCapability(capability, facing);
     }
+
+
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)